src/store/main/sagas/loadReadOnlyData.ts

```diff
@@ -15,7 +15,7 @@ export function* loadReadOnlyData() {
       coingeckoPrices,
       isLoading: false,
     }));
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
Can we use `error` type instead of `any` 🙇🏻

Reply:
Done 0357287